src/Symfony/Bundle/FrameworkBundle/Tests/Functional/Bundle/TestBundle/AutowiringTypes/AnnotationReader.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\AutowiringTypes;
+
+use Doctrine\Common\Annotations\Reader;
+
+class AnnotationReader
```

Comment:
Maybe can we create one test class where we will inject all services having an `autowiring_type` entry in their definition?

It will ease adding tests for other autowired services.